src/native/linux/qgslinuxnative.cpp

```diff
@@ -102,6 +102,22 @@ void QgsLinuxNative::hideApplicationProgress()
   QDBusConnection::sessionBus().send( message );
 }
 
+void QgsLinuxNative::setApplicationBadgeCount( int count )
+{
+  // the badge will only by shown when the count is greater than one
```

Comment:
I'd move this requirement up to app - regardless of the platform, we wouldn't want to show a badge count of 1

Reply:
@nyalldawson , didn't we want to show a bade count of 1 on OS X since it doesn't have a progress indicator?

Reply:
typo: s/by/be